### PR TITLE
fix(cli): add redis to facade

### LINKED ISSUE #792: @sourceloop/cli generated facade has a missing redis datasource
**Describe the bug**
@sourceloop/cli generated facade has a missing redis datasource

**To Reproduce**
1. Generate a facade using sourceloop/cli
2. Generated facade fails to run throwing an error about missing datasource

**Expected behavior**
A default datasource should be provided

### PR DESCRIPTION
**Pull request by AnkurBansalSF**
## Description

added a redis datasource to facades.

Fixes #792 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by yeshamavani — CHANGES_REQUESTED** [review]

File: packages/cli/src/generators/microservice/templates/src/application.ts.ejs [hunk 1]
```diff
@@ -188,6 +188,7 @@ export class <%= project.applicationName %> extends BootMixin(RestApplication) {
     });
 
     this.component(RestExplorerComponent);
+    console.log('my changes');
```
[the remark this answers is not in the document]
Comment (AnkurBansalSF) [reply packages/cli/src/generators/microservice/templates/src/application.ts.ejs]: done

File: packages/cli/src/generators/microservice/index.ts [hunk 2]
```diff
@@ -409,6 +411,20 @@ export default class MicroserviceGenerator extends AppGenerator<MicroserviceOpti
...
+    if (this.projectInfo.facade) {
+      const nameArr = ['redis'];
+      this.fs.copyTpl(
+        this.templatePath(REDIS_DATASOURCE),
+        this.destinationPath(join('src', 'datasources', 'redis.datasource.ts')),
+        {
+          project: this.projectInfo,
+        },
+      );
+      nameArr;
```
[the remark this answers is not in the document]
Comment (AnkurBansalSF) [reply packages/cli/src/generators/microservice/index.ts]: fixed

File: packages/cli/src/generators/microservice/index.ts [hunk 3]
```diff
@@ -409,6 +411,26 @@ export default class MicroserviceGenerator extends AppGenerator<MicroserviceOpti
     }
   }
 
+  private _createFacadeRedisDatasource() {
+    if (this.projectInfo.facade) {
```
Comment (akshatdubeysf) [inline packages/cli/src/generators/microservice/index.ts:415]: this.options.facade?

Comment (AnkurBansalSF) [reply packages/cli/src/generators/microservice/index.ts]: done

**Review by akshatdubeysf — APPROVED** [review]

**Pull request merged**